packages/core/src/app/checkout/getCheckoutStepStatuses.ts

```diff
@@ -30,12 +30,26 @@ const getCustomerStepStatus = createSelector(
                 : false;
         const isGuest = !!(customer && customer.isGuest);
         const isComplete = hasEmail || isUsingWallet;
+        const isEditable = isComplete && !isUsingWallet && isGuest
+
+        if (sessionStorage.getItem('stripeLink') != null && hasEmail) {
+            if (customer?.isStripeLinkAuthenticated === undefined) {
+                sessionStorage.setItem('stripeLink', 'customerReloaded')
```

Comment:
After consulting with the security engineer, we think it is better to avoid using `sessionStorage` .

Can we build a helper function to check the state of reloading?